test/e2e/positive.go

```diff
@@ -269,6 +269,56 @@ var _ = ginkgo.Describe("[PositiveFlow] Deployer execution", func() {
 			e2epods.WaitForPodToBeRunning(cli, testPod.Namespace, testPod.Name)
 		})
 	})
+	ginkgo.Context("with a running cluster with NFD as the topology-updater component", func() {
+		ginkgo.BeforeEach(func() {
+			err := deploy([]string{"--updater-type", "NFD"})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.AfterEach(func() {
+			err := remove([]string{"--updater-type", "NFD"})
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.It("should have some information in the noderesourcetopology object", func() {
```

Comment:
This part is a duplicate of https://github.com/k8stopologyawareschedwg/deployer/blob/main/test/e2e/positive.go#L193
but this is only a testing code, so it's not so bad, but I'm open to suggestions.

Reply:
duplicating test code is actually ok: https://about.sourcegraph.com/go/advanced-testing-in-go/#Golden-files-also-test-flags